step-ca/README.mdx

```diff
@@ -88,3 +88,22 @@ For strong protection of your CA signing keys, we've built [`step-ca` integratio
 ### Other Integrations
 
 `step-ca` plays well with Kubernetes cert-manager and Envoy Secret Discovery Service. For more information, see [Integrations](./integrations.mdx).
+
+## Limitations
+
+While we think `step-ca` is the best open source, scalable online Certificate Authority on the internet, every piece of software comes with limitations and tradeoffs.
+
+`step-ca` is designed to favor a simple deployment of a scalable two-tiered X.509 PKI, with one Root CA and one Intermediate CA that issues end-entity certificates with passive revocation. Its SSH support is favors having two single-tier SSH CAs—a single SSH Host CA, and a single SSH User CA.
+
+Here are some limitations of `step-ca` that grew out of our design choices:
+- It issues X.509 certificates from a single configured Intermediate CA; multiple issuing CAs are not supported
+- Its root CA is always offline; a single-tier PKI is not supported
```

Comment:
It might be possible by configuring the root as the intermediate too. Haven't tested that, I think. But I guess it's fair to put it like you did: no official support.